### ISSUE TITLE
git-gutter-fringe isn't dispaly color on the fringe. when i use othe theme

### ISSUE THREAD

**Issue by meziberry** [radian-software/radian#456]
I turn off the default theme, then using the doom-theme. but the `git-gutter-fringe` can't work correctly.  if i use the default theme `zerodark` , it is ok.  I think the reason is about the advice `radian--advice-git-gutter-remove-bitmaps`  I was trying to fix it, but i don't understand how it is working.

Thank you for your share.
Emacs version: `28.0.50`
OS: `windows 10`

**Comment by raxod502** [comment]
Does the problem occur with every other theme, or only with your specific theme? You can test to see if the problem is the advice by disabling it and checking if that changes anything:

```elisp
(advice-remove #'git-gutter-fr:view-diff-infos #'radian--advice-git-gutter-remove-bitmaps)
```

**Comment by meziberry** [comment]
I test on the theme of `doom-themes zenburn-theme  leuven-theme`, the color of fringe is still disappeared.  Evenly after adding `(advice-remove #'git-gutter-fr:view-diff-infos #'radian--advice-git-gutter-remove-bitmaps)`

It is strange that the gutter-fringe is coming, when i comment out the following code in `radian.el`
```emacs-lisp
    (radian-defadvice radian--advice-git-gutter-remove-bitmaps
        (func &rest args)
      :around #'git-gutter-fr:view-diff-infos
      "Disable the cutesy bitmap pluses and minuses from `git-gutter-fringe'.
Instead, display simply a flat colored region in the fringe."
      (radian-flet ((defun fringe-helper-insert-region
                        (beg end _bitmap &rest args)
                      (apply fringe-helper-insert-region
                             beg end 'radian--git-gutter-blank args)))
        (apply func args))) 
```
* the style of fringe after comment out `radian-defadvice radian--advice-git-gutter-remove-bitmaps`
![image](https://user-images.githubusercontent.com/10571785/82767094-601bcf00-9e57-11ea-8e11-e8fb82541baf.png)
* the image of fringe with problem
![image](https://user-images.githubusercontent.com/10571785/82767193-3e6f1780-9e58-11ea-8b73-8f8b13e6c8dc.png)

**Comment by raxod502** [comment]
Thank you for the report. I believe the problem is fixed now. The problem was that Zerodark defined a background color for the face, while other themes did not.

**Comment by meziberry** [comment]
Now, It's normally displayed on `lenven-theme`.  but on `doom-theme  zenburn-theme` it is still invisible.

**Comment by raxod502** [comment]
I can confirm that that is still a problem. I looked at it, and was unable to figure out what's going on. The face is customized correctly, however for some reason when the `doom-one` theme is enabled the background of the fringe bitmap is never shown. I checked to see if the problem was the customization of the `fringe` face, but no dice.

**Comment by raxod502** [comment]
This seems fixed to me now with Emacs 28.1:

![image](https://user-images.githubusercontent.com/6559064/221753038-4720361f-33c7-40e3-8fac-81a8bd8fbc31.png)
